### PR TITLE
fix(@angular-devkit/build-angular): maxModules were deprecated and renamed to modulesSpace

### PR DESCRIPTION
**Pull request by valorkin**
maxModules were deprecated and renamed to modulesSpace

### CODE REVIEW

**Review by alan-agius4 — APPROVED** [review]
LGTM

File: packages/angular_devkit/build_angular/src/webpack/configs/stats.ts [hunk 1]
```diff
@@ -41,6 +41,12 @@ const verboseWebpackOutputOptions = {
   logging: 'verbose',
 };
 
+if (!isWebpackFiveOrHigher()) {
```
Comment (alan-agius4) [inline packages/angular_devkit/build_angular/src/webpack/configs/stats.ts:44]: Nit: the not/! here can be removed if the logic below is inverted.

```
if (isWebpackFiveOrHigher()) {
  verboseWebpackOutputOptions['modulesSpace'] = Infinity;
} else {
  verboseWebpackOutputOptions['maxModules'] = Infinity;
}
```

File: packages/angular_devkit/build_angular/src/webpack/configs/stats.ts [hunk 2]
```diff
@@ -25,11 +26,10 @@ const webpackOutputOptions = {
   moduleTrace: false,
 };
 
-const verboseWebpackOutputOptions = {
+const verboseWebpackOutputOptions: {[index: string]: boolean|string|number} = {
```
Comment (alan-agius4) [inline packages/angular_devkit/build_angular/src/webpack/configs/stats.ts:29]:
```suggestion
const verboseWebpackOutputOptions: Record<string, boolean | string | number> = {
```

File: packages/angular_devkit/build_angular/src/webpack/configs/stats.ts (hunk 1 above)
Comment (valorkin) [reply packages/angular_devkit/build_angular/src/webpack/configs/stats.ts]: done

File: packages/angular_devkit/build_angular/src/webpack/configs/stats.ts (hunk 2 above)
Comment (valorkin) [reply packages/angular_devkit/build_angular/src/webpack/configs/stats.ts]: applied

**Review by alan-agius4 — APPROVED** [review]

**Comment by valorkin** [comment]
thanks

**Pull request merged**